server/bundles/io.cloudbeaver.service.fs/src/io/cloudbeaver/service/fs/model/WebFSServlet.java

```diff
@@ -98,7 +98,7 @@ private void doPost(WebSession session, HttpServletRequest request, HttpServletR
                 }
             }
         } catch (Exception e) {
```

Comment:
Why use e.getCause()? It may produce NPE.

Use both messages: "File upload filed: " + e.getMessage()

If you need original error use GeneralUtils.getRootCause